### PR TITLE
Create step dataset-is-valid

### PR DESCRIPTION
**Pull request by severo**
See https://github.com/huggingface/datasets-server/issues/891

### CODE REVIEW

File: services/worker/src/worker/job_runners/dataset/is_valid.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+]
+FIRST_ROWS_KINDS = ["split-first-rows-from-streaming", "split-first-rows-from-parquet"]
+
+
+def compute_is_valid_response(dataset: str) -> Tuple[DatasetIsValidResponse, float]:
+    """
+    Get the response of /is-valid for one specific dataset on huggingface.co.
+
+    A dataset is valid if:
+    - /splits is valid
```
Comment (albertvillanova) [inline services/worker/src/worker/job_runners/dataset/is_valid.py:35]: One question:
```suggestion
    - /splits is valid for at least one of the splits 
```

**Review by albertvillanova — APPROVED** [review]
Thanks.

File: services/worker/src/worker/job_runners/dataset/is_valid.py (hunk 1 above)
Comment (severo) [reply services/worker/src/worker/job_runners/dataset/is_valid.py:35]: You're right. (nit: configs, not splits)

Comment (severo) [reply services/worker/src/worker/job_runners/dataset/is_valid.py]: done

**Pull request merged**